app-admin/ansible-cmdb/ansible-cmdb-1.30-r1.ebuild

```diff
@@ -0,0 +1,24 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_9 )
+
+inherit distutils-r1
+
+DESCRIPTION="Ansible Configuration Management Database"
+HOMEPAGE="https://github.com/fboender/ansible-cmdb"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+DEPEND="
+	dev-python/setuptools[${PYTHON_USEDEP}]"
```

Comment:
setuptools can be removed from DEPEND, it is maintained correctly by the `distfiles-r1.eclass`
```suggestion
```